docs/docs/guides/tools/comparison.md

```diff
@@ -0,0 +1,113 @@
+# Comparison
+
+The Weave Comparison feature allows you to visually compare and diff code, traces, prompts, models, model configs, evaluation metrics, and evaluation predictions.  You can compare two objects side-by-side or analyze a larger set of objects to identify differences, patterns, and trends.
```

Comment:
Evaluations have their own custom comparison page so while you can compare them (as they are traces) using this page, I might omit "evaluation metrics, and evaluation predictions" from this intro to avoid confusing users about what we'd recommend.

Reply:
Makes sense, removed in https://github.com/wandb/weave/pull/3124/commits/65d43eb765868268559f8573347bdcc4a3302dcd